lms/djangoapps/courseware/models.py

```diff
@@ -145,10 +153,11 @@ def __unicode__(self):
         return unicode(repr(self))
 
 
-class StudentModuleHistory(models.Model):
+class StudentModuleHistory(CallStackMixin, models.Model):
```

Comment:
Arguments about multiple inheritance aside python is depth first, left to right when it comes to searching in multiple inheritance. It's quite possible that adding in CallStackMixin before models.Model could have unintended impacts on any downstream classes that may inherit from StudentModuleHistory.